notebook/static/notebook/js/textcell.js

```diff
@@ -291,6 +291,7 @@ define([
                 // links in markdown cells should open in new tabs
                 html.find("a[href]").not('[href^="#"]').attr("target", "_blank");
                 that.set_rendered(html);
+                that.element.find('div.text_cell_render').find('div').addClass('html_rendered');
```

Comment:
I appreciate the patience. My thinking was that since output areas are 
apparently just repurposed rather than destroyed and recreated, it was 
better to respond to whatever changes the user may have made and declare 
the divs as rendered_html when rendering them.

---
Regards,
Nitin Dahyabhai




From:   Min RK <notifications@github.com>
To:     jupyter/notebook <notebook@noreply.github.com>
Cc:     Nitin Dahyabhai/Raleigh/IBM@IBMUS
Date:   07/21/2015 06:29 PM
Subject:        Re: [notebook] Issue 158 - Rendered markdown div should 
get .rendered_html class (#216)



In notebook/static/notebook/js/textcell.js:
> @@ -291,6 +291,7 @@ define([
>                  // links in markdown cells should open in new tabs
>                  html.find("a[href]").not('[href^="#"]').attr("target", 
"_blank");
>                  that.set_rendered(html);
> + 
that.element.find('div.text_cell_render').find('div').addClass('html_rendered');

I still think this isn't the right place to do it, and the class is 
rendered_html, not html_rendered. Look at where the html output adds the 
class (append_html), and there is a similar place in the markdown output 
(append_markdown) where it can be added, without making any jquery calls.
—
Reply to this email directly or view it on GitHub.